src/main/java/com/marklogic/kafka/connect/source/RowBatcherSourceTask.java

```diff
@@ -145,10 +148,18 @@ private void configureRowBatcher(Map<String, Object> parsedConfig, RowBatcher<Js
         }
 
 
-        String dslPlan = (String) parsedConfig.get(MarkLogicSourceConfig.DSL_PLAN);
-        RowManager rowMgr = rowBatcher.getRowManager();
-        RawQueryDSLPlan plan = rowMgr.newRawQueryDSLPlan(new StringHandle(dslPlan));
-        rowBatcher.withBatchView(plan);
+        Boolean configuredForDsl = checkForNonEmptyString((String) parsedConfig.get(MarkLogicSourceConfig.DSL_QUERY));
```

Comment:
To keep `configureRowBatcher` to a manageable size, move this to a `private void configureBatchView(rowBatcher, parsedConfig)` method - which could be static too, if you want to start trying out that pattern to see how it looks.

Reply:
Is this an accidental repeat, or is there a difference between this one and the 2nd conversation from the top?